src/WaterLily.jl

```diff
@@ -75,29 +75,31 @@ scales.
 sim_time(sim::Simulation) = time(sim)*sim.U/sim.L
 
 """
-    sim_step!(sim::Simulation,t_end;max_steps=typemax(Int),remeasure=true,verbose=false)
+    sim_step!(sim::Simulation,t_end=sim(time)+Δt;max_steps=typemax(Int),remeasure=true,verbose=false)
 
 Integrate the simulation `sim` up to dimensionless time `t_end`.
 If `remeasure=true`, the body is remeasured at every time step.
 Can be set to `false` for static geometries to speed up simulation.
 """
 function sim_step!(sim::Simulation,t_end;max_steps=typemax(Int),verbose=false,remeasure=true)
```

Comment:
The change in the docs here (`t_end=sim(time)+Δt`) is not reflected in the actual code?

Reply:
If you don't supply an end time, the code on line 90 runs directly, instead of this wrapper. So exactly one time step is taken, as indicated in the docs.

Reply:
yup, missed the multiple dispatch.